### DISCUSSION TITLE
Is using the beta version logical for a production ready app? When will the stable version be released?

### DISCUSSION THREAD

**Discussion by MobinAskari** [nextauthjs/next-auth#10623] [Help]
Basically what the title says, I understand the meaning of word the "beta", but I was wondering if the development process of (Next-Auth@beta) is close to a stable release and it's logical to adapt it in a production ready app.

Also when is it expected for the stable version to be released? A date range is sufficient, too, if there's not a targeted date

**Comment by mid2ly** [comment]
Strictly speaking, stable versions are more secure than beta versions, but beta versions can generally be considered safe enough.

However, because the beta version is released more than once a day (at least at a faster rate than the stable version), there may come a situation where code written yesterday needs to be changed today.

This can be fatal in production, **but if you're willing to risk it**, there's nothing wrong with using it in production.

**Comment by ebratz** [comment]
Just wondering if there are any roadmap/tests status publicly available for v5 stable release. What % of the things are done right now?
That would help a ton in order for new projects to decide if it worth waiting before moving to v5.
Thanks in advance

**Comment by codigoderey** [comment]
> Basically what the title says, I understand the meaning of word the "beta", but I was wondering if the development process of (Next-Auth@beta) is close to a stable release and it's logical to adapt it in a production ready app.
> 
> Also when is it expected for the stable version to be released? A date range is sufficient, too, if there's not a targeted date

If the beta version works for your need is ok to use it, just have in mind that if the stable version has breaking changes you will have to make those changes and test before pushing to production.
